src/test/java/bio/terra/janitor/db/JanitorDaoTest.java

```diff
@@ -83,33 +81,14 @@ public void serializeCloudResourceUid() {
   public static Map<String, Object> queryTrackedResource(
       NamedParameterJdbcTemplate jdbcTemplate, CloudResourceUid resourceUid) {
     String sql =
-        "SELECT id, resource_uid::text, resource_type, creation, expiration, state FROM tracked_resource WHERE resource_uid::jsonb @> :resource_uid::jsonb";
+        "SELECT id, resource_uid::text, resource_type, creation, expiration, state " +
+                "FROM tracked_resource " +
+                "WHERE resource_uid::jsonb @> :resource_uid::jsonb";
```

Comment:
does this work with '=' instead of '@>'?

Reply:
Yup!